README.md

```diff
@@ -3,6 +3,8 @@
 ![](https://github.com/linkml/linkml-runtime/workflows/Build/badge.svg)
 [![badge](https://img.shields.io/badge/launch-binder-579ACA.svg)](https://mybinder.org/v2/gh/linkml/linkml-runtime/main?filepath=notebooks)
 [![PyPi](https://img.shields.io/pypi/v/linkml-runtime.svg)](https://pypi.python.org/pypi/linkml)
+[![PyPIDownloadsTotal](https://pepy.tech/badge/linkml-runtime)](https://pepy.tech/project/linkml-runtime)
+[![PyPIDownloadsMonth](https://img.shields.io/pypi/dm/scanpy?logo=PyPI&color=blue)](https://pypi.org/project/linkml-runtime)
```

Comment:
Quick question about where these download statistics are coming from? From pepy? Clicking on this badge just takes us to the linkml-runtime repo on PyPI? Perhaps the link should be: https://pepy.org/project/linkml-runtime